cmd/bootstrap/utils/key_generation.go

```diff
@@ -228,6 +229,35 @@ func WriteMachineAccountFiles(chainID flow.ChainID, nodeInfos []bootstrap.NodeIn
 	return nil
 }
 
+func WriteMachineAccountFile(
+	chainID flow.ChainID,
+	nodeID flow.Identifier,
+	accountAddress flow.Address,
+	accountKey encodable.MachineAccountPrivKey,
+	write WriteJSONFileFunc) error {
+	// ensure the chain ID is for a transient chain, where it is possible to
+	// infer machine account addresses this way
+	if !chainID.Transient() {
+		return fmt.Errorf("cannot write default machine account files for non-transient network")
+	}
```

Comment:
This isn't really necessary. We aren't inferring the machine account addresses here, since it's passed in as an argument.
```suggestion
```